### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/837d42759246700a10d18e3bb38912fd06b062d2

### CODE REVIEW

File: en/brands.ftl [hunk 1]
```diff
@@ -13,7 +13,6 @@
 -brand-name-facebook = Facebook
 -brand-name-github = GitHub
 -brand-name-google = Google
--brand-name-hulu = Hulu
```
Comment (peiying2) [inline en/brands.ftl:16]: Why is this removed? The brand term was added in the adblocker.ftl file

**Review by peiying2 — CHANGES_REQUESTED** [review]
@craigcook Hulu was added due to reference in the adblocker.ftl file which was migrated this week.

File: en/brands.ftl (hunk 1 above)
Comment (craigcook) [reply en/brands.ftl:16]: It wasn't in my PR. Investigating...

Comment (craigcook) [reply en/brands.ftl:16]: Oh I see, the file in this repo is ahead of the one in bedrock so the robot thought this line was deleted (since it's not in bedrock yet). Hulu is being added in https://github.com/mozilla/bedrock/pull/9439/ that hasn't merged yet, but I'll review that now and it will generate a new PR when it merges.

Comment (craigcook) [reply en/brands.ftl:16]: So... that PR had `[skip l10n]` so I don't think the robot will make an automated PR to replace this one. I think now the only way to fix this is to manually open a replacement PR, so I'll do that.

Comment (craigcook) [reply en/brands.ftl:16]: Oh hang on, I can push to this branch. (sorry for all the comments, just talking to myself...)

Comment (peiying2) [reply en/brands.ftl:16]: Really appreciate you taking the time to get  to the bottom of this.

**Comment by craigcook** [comment]
Fixed brands.ftl, sorry bout that. Repos fell out of sync.

**Review by peiying2 — APPROVED** [review]
Thank you for the fix. Looks good!
